networkdb/delegate.go

```diff
@@ -402,6 +403,7 @@ func (d *delegate) NotifyMsg(buf []byte) {
 	d.nDB.handleMessage(buf, false)
 }
 
+// XXX: should this limit be shared?
 func (d *delegate) GetBroadcasts(overhead, limit int) [][]byte {
```

Comment:
Not sure why this PR(https://github.com/docker/libnetwork/pull/1446) changed it this way. To me makes more sense the original code with the simple return...